R/warning-utils.R

```diff
@@ -0,0 +1,35 @@
+# Source constants
```

Comment:
Can you format this file better?
1. Add new lines between the functions
2. Add comments for each function
3. Rename `ivalid_cat_link` to `invalid_cat_link`
4. Can you use better grammer in the warnings messages? Full stops and capitalization for example.